### ISSUE TITLE
Large numeric values are rounded and double fields are returned as scientific notation

### ISSUE THREAD

**Issue by oruchreis** [Restream/reindexer#87]
Hi,
I have a double tree index field has this value:
`638692992000000000`
I can also see this value correctly in the face interface in this form.
But when I query this index via grpc and get it as json, it returns as scientific notation as follows in json:
```json
{
    "Date": 6.38692992e+17
}
```
However, this problem occurred after I upgraded Reindexer to version 3.30. There was no problem with version 3.14 before that.

Best Regards.

**Comment by oruchreis** [comment]
I think the gason lib is used on the c++ side. i saw that after a certain version of gason it became standard this behavior, so i solved it on my side by deserialising the scientific notation. so i close this issue.

**Comment by oruchreis** [comment]
I think another problem is that the reindexer seems to round off data of a certain size. When I tried to set this value to various index combinations from Face UI, I saw that the value was rounded. I think either when sending the document from Face UI or when Reindexer receives and processes the document, it rounds the values. This may be the source of the problem I'm having, i.e. on the json serialiser/deserialiser side of Reindexer.
![image](https://github.com/user-attachments/assets/777ebc45-c01f-4629-8aef-151806c3ad3b)
![image](https://github.com/user-attachments/assets/b0438caf-b873-40e3-8a75-36397ef06d26)
![image](https://github.com/user-attachments/assets/6258d00e-e8a8-4cf0-b3c8-4d6756c6daae)
